ruby/vampires.rb

```diff
@@ -0,0 +1,61 @@
+# Vampire Detector Quiz
+
+# release 4 loop
+puts 'How many are we interviewing?'
+applicants = Integer(gets.chomp)
+
+until applicants.zero?
+
+  # The Quiz
+  puts "Welcome to Werewolf Inc. What's your name?"
+  name = gets.chomp
+  vampire = false
+  vampire = true if name == 'Drake Cula' || name == 'Tu Fang'
+
+  puts 'How old are you?'
+  age_range = Integer(gets.chomp)
+  puts 'What year were you born?'
+  year_number = Integer(gets.chomp)
+  age = false
+  age = true if age_range + year_number == 2017
+
+  puts 'Our company cafeteria serves garlic bread. Should we order some for you?'
+  garlic_preference = gets.chomp
+  garlic = false
+  garlic = true if garlic_preference == 'yes' || garlic_preference == 'y'
+
+  puts "Would you like to enroll in the company's health insurance?"
+  insurance_status = gets.chomp
+  insurance = false
+  insurance = true if insurance_status == 'yes' || insurance_status == 'y'
+
+  # Allergy loop
+  puts "Now to ask if you're allergic to anything"
+  allergies = ''
+  while allergies != 'done'
```

Comment:
`while !=` == `until ==` 

Personal preference is to avoid negative conditions.